### PR TITLE
docs: transpile warning added for plugins

### PR DESCRIPTION
**Pull request by debs-obrien**

### CODE REVIEW

**Comment by debs-obrien** [comment]
With ref to this pr: https://github.com/nuxt/nuxtjs.org/pull/840 which was made to old docs. As we didn't have an example that uses the axios import have added that as well as comment in pr to old docs

File: content/en/guides/directory-structure/plugins.md [hunk 1]
```diff
@@ -129,13 +129,41 @@ Then we can use it directly in your page components:
...
+<base-alert type="info">
+
+If you get an _Cannot use import statement outside a module_ error, you may need to add your package to the `build` > `transpile` option in `nuxt.config.js` for webpack loader to make your plugin available.
+
+</base-alert>
+
+```js{}[nuxt.config.js]
+build: {
+  // You can extend webpack config here
+  transpile: ['your-plugin-name'],
```
Comment (pi0) [inline content/en/guides/directory-structure/plugins.md:163]: `npm-package-name` :)

File: content/en/guides/directory-structure/plugins.md [hunk 2]
```diff
@@ -129,13 +129,41 @@ Then we can use it directly in your page components:
...
 
+Another way to use `axios` without installing the module is by importing `axios` direct in the `<script>` tag.
+
+```js{}[pages/index.vue]
+<script>
+import axios from 'axios'
+
+export default {
+	async asyncData ({ params }) {
+	  const  post  = await axios.$get(`https://api.nuxtjs.dev/posts/${params.id}`)
```
Comment (pi0) [inline content/en/guides/directory-structure/plugins.md:147]: Directly using `axios`, it does not support `$` utils

Comment (debs-obrien) [reply content/en/guides/directory-structure/plugins.md:147]: oops. - pr: https://github.com/nuxt/nuxtjs.org/pull/884

**Comment by debs-obrien** [comment]
https://github.com/nuxt/nuxtjs.org/pull/884

**Pull request merged**